code/modules/effects/map_effects/airlock_marker.dm

```diff
@@ -0,0 +1,135 @@
+
+/obj/effect/map_effect/marker
+	name = "map marker parent abstract object"
+	icon = 'icons/effects/map_effects.dmi'
+	icon_state = "map_marker"
+
+/obj/effect/map_effect/marker_helper
+	name = "map marker helper parent abstract object"
+	icon = 'icons/effects/map_effects.dmi'
+	icon_state = "map_marker"
+
+/// map of airlock marker `id_tag` to a list of map_effect/airlock_markers
+var/global/list/airlock_markers = list()
```

Comment:
global vars should generally be at the start of the file

Reply:
outdated/refactored